Raven.Database/Server/Controllers/StreamsController.cs

```diff
@@ -246,69 +246,67 @@ public async Task<HttpResponseMessage> Exploration(string collection)
 
             var msg = GetEmptyMessage();
 
-            using (var cts = new CancellationTokenSource())
+            var cts = new CancellationTokenSource();
+            var timeout = cts.TimeoutAfter(TimeSpan.FromSeconds(timeoutSeconds));
```

Comment:
We are also seems to be leaking a timer here

Reply:
This is handled by `StreamQueryContent`, if there is an error here that isn't caught, we'll let the finalizer handle this.
Expected to be very rare